_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -14,6 +15,14 @@
 
 
 // ----------------------------
+// #PATHS
+// ----------------------------
+$static-path: '..' !default;
+$font-path: '#{$static-path}/fonts' !default;
+$image-path: '#{$static-path}/images' !default;
+
```

Comment:
Yeah, the thought might be that someone may keep their static assets in a different structure, so in that case, they could just change one variable rather than syncing two up. I can go either way on this. How strongly do you feel about simplifying it to 2 vars?